### ISSUE TITLE
wordpress module does not work with wordpress 2.1 and 2.2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Daniel3356/recaptcha#2]
```
What steps will reproduce the problem?
1. activate module
2. set up public and private keys
3. post a comment without typing anything in the captcha box

What is the expected output? What do you see instead?
It should block the comment but it does not. the comment will just show on
the site with along with an error message: WP Error:incorrect-captcha-sol

What version of the product are you using? On what operating system?
using recaptcha version 2.3 on wordpress 2.1.2 and wordpress 2.2.0




```

Original issue reported on code.google.com by `mihai.se...@gmail.com` on 25 Jun 2007 at 10:58

**Comment by GoogleCodeExporter** [comment]
```
same problem here with recaptcha version 2.3 and wordpress 2.2.1. Is that 
somehow
related to wordpress' own comment and spam settings?
```

Original comment by `markus.r...@gmail.com` on 27 Jun 2007 at 2:01

**Comment by GoogleCodeExporter** [comment]
```
This seems to be partially solved in recaptcha-wordpress-2.5.zip, using 
Wordpress 
2.2. If I do not type anything, the comment will not be posted, and one has to 
try 
again.

However, it seems it suffice to type only the first of the two words. I don't 
know 
if the first word always is the "known" word, so this might depend upon that. 

Also, this might just be a feature - spam will be stopped anyway I guess.
```

Original comment by `grons...@gmail.com` on 6 Jul 2007 at 7:30

**Comment by GoogleCodeExporter** [comment]
```
Not working here! :(

Running Wordpress 2.2.1 and Recaptcha 2.5.

The form shows up on my comment page but you can still comment without even 
entering
a single char into the Recaptcha. :(
```

Original comment by `war59312` on 20 Jul 2007 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
The WP Error:incorrect-captcha-sol issue was fixed with 2.5
```

Original comment by `ben.maurer` on 4 Aug 2007 at 12:44
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `adrian.g...@gmail.com` on 30 Mar 2012 at 6:19
- Changed state: **Verified**
